tests/unit/moduleapi/getkeys.tcl

```diff
@@ -86,4 +86,15 @@ start_server {tags {"modules"}} {
     test "Unload the module - getkeys" {
         assert_equal {OK} [r module unload getkeys]
     }
+
+    test {KEYS command return expired keys when allow_access_expired is 1} {
```

Comment:
why not move this test to hash.tcl?
please don't use force push, which is hard to review. thx.